### PR TITLE
wr-192: Add hoverable tooltips on prompt tags

### PR DESCRIPTION
**Pull request by IrynaK-a**
image

### CODE REVIEW

File: frontend/src/pages/articles/components/tags/tags.tsx [hunk 1]
```diff
@@ -13,7 +16,20 @@ const Tags: React.FC<Properties> = ({ tags }) => (
           <span className={styles.categoryText}>{tag.category}</span>
         </div>
         <div className={styles.prompt}>
-          <p className={styles.promptText}>{tag.text}</p>
+          <BlockWithTooltip
+            tooltipContent={tag.text}
+            placement="bottom"
+            hasToShowTooltip={true}
```
Comment (andanf-e) [inline frontend/src/pages/articles/components/tags/tags.tsx]: It's `true`  by default
```suggestion

```

File: frontend/src/pages/articles/components/tags/tags.tsx [hunk 2]
```diff
@@ -13,7 +16,20 @@ const Tags: React.FC<Properties> = ({ tags }) => (
...
+            tooltipId={DataTooltipId.PROMPT_TAG_TOOLTIP}
+          >
+            <p className={styles.promptText}>{tag.text}</p>
+          </BlockWithTooltip>
+          <Tooltip
+            id={DataTooltipId.PROMPT_TAG_TOOLTIP}
+            shouldHideArrow={true}
```
Comment (andanf-e) [inline frontend/src/pages/articles/components/tags/tags.tsx]:
```suggestion
            shouldHideArrow
```

**Review by andanf-e — CHANGES_REQUESTED** [review]
Provide screenshots

File: frontend/src/pages/articles/components/tags/tags.tsx [hunk 3]
```diff
@@ -13,7 +16,19 @@ const Tags: React.FC<Properties> = ({ tags }) => (
...
+            placement="bottom"
+            tooltipId={DataTooltipId.PROMPT_TAG_TOOLTIP}
+          >
+            <p className={styles.promptText}>{tag.text}</p>
+          </BlockWithTooltip>
+          <Tooltip
+            id={DataTooltipId.PROMPT_TAG_TOOLTIP}
+            shouldHideArrow
+            className={styles.promptTagTooltip}
+            offset={5}
```
Comment (andanf-e) [inline frontend/src/pages/articles/components/tags/tags.tsx]: Check **B31**

Comment (IrynaK-a) [reply frontend/src/pages/articles/components/tags/tags.tsx]: sorry, what does that mean? I don't get it.

Comment (artemmatiushenko1) [reply frontend/src/pages/articles/components/tags/tags.tsx]: https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md

Comment (IrynaK-a) [reply frontend/src/pages/articles/components/tags/tags.tsx]: Of course, it's quality criteria:grimacing:
Thanks!

**Review by andanf-e — DISMISSED** [review]

**Review by andanf-e — APPROVED** [review]

**Review by NazarKovtun1 — APPROVED** [review]

**Pull request merged**